### DISCUSSION TITLE
Is the IATP compatible to OIDC4VC?

### DISCUSSION THREAD

**Discussion by efiege** [eclipse-tractusx/tractusx-edc#1119] [Q&A] [answered]
[In the IATP Specification](https://github.com/eclipse-tractusx/identity-trust/blob/1341a0661febda494d0c5c8cc5e262d6344ea66a/specifications/credential.issuance.protocol.md?plain=1#L17-L21) it has been mentioned, that:

> This specification draws from the
[OpenID for Verifiable Credential Issuance specification](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-introduction)
but differs in its focus on service-to-service interactions where end-user devices are not involved in the
protocol flow. Moreover, the current specification accommodates the requirement for long-running interactions typically
associated with manual workflows that are best modelled using asynchronous messaging paradigms.

As such there seem to be some alignements, but also differences and the following questions arose:
- Which are the differences between IATP and OIDC4VC?
- Is the IATP compatible to OIDC4VC? Or is it compatible to parts of OIDV4VC?
  - Can the MIW-Extensions be used with OIDC4VC Components?
- If it is not compatible: Is this planned for the future?

I hope this is the right place for this question, also thanks in advance for the answers!

**Comment by jimmarino** [comment, marked as answer]
I'm not sure what "compatible" implies. If you are asking whether IATP supports OID4VCI APIs and flows, the answer is no for the reasons laid out in the quoted section.
